packages/deploy-utils/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "baseUrl": "src",
+    "module": "CommonJS",
+    "moduleResolution": "Node",
+    "paths": {
+      "@evg-ui/lib/*": ["../../lib/src/*"]
+    },
+    "types": [
+      "vitest/globals",
+      "vitest/jsdom"
+    ]
```

Comment:
I was wondering if these overrides not get inherited from the base tsconfig? :0 (excluding the `paths` field)